package.json

```diff
@@ -124,8 +124,7 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "@mapbox/mr-ui": "^0.10.0",
-    "@mapbox/mbx-assembly": "^0.31.0",
```

Comment:
The tests/peer-deps.test.js is failing because it expects to see mr-ui as a peerDependency. We'll want to add it back and set the version to `0.12.0`. This will help assert that all site repos use this same mr-ui version.